scripts/set-default-credentials.sh

```diff
@@ -9,10 +9,14 @@ if [ "${TRAVIS}" != "true" ]; then
   export TRAVIS_REPO_SLUG="firebase/firebase-tools"
 fi
 
-GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-private.json"
-if [ "${TRAVIS_REPO_SLUG}" == "firebase/firebase-tools" ]; then
-  GOOGLE_APPLICATION_CREDENTIALS="${CWD}/scripts/creds-public.json"
+if [ -z "${LOCAL}" ]; then
```

Comment:
Since we're using bash we can do
```
if [[ -z $LOCAL ]]; then
```

The [double-brackets](https://stackoverflow.com/questions/669452/is-double-square-brackets-preferable-over-single-square-brackets-in-ba) are slightly more ergonomic to use.